packages/core/src/plugins/__tests__/SegmentDestination.test.ts

```diff
@@ -72,14 +76,57 @@ describe('SegmentDestination', () => {
       integrations: {},
     };
 
-    const expectedIntegrations = {
-      firebase: false,
+    const result = plugin.execute(event);
+    expect(result).toEqual({
+      ...event,
+      _metadata: {
+        bundled: ['firebase'],
+        unbundled: [],
+        bundledIds: [],
+      },
+    });
+  });
+
```

Comment:
here is a more detailed test for the destination metadata plugin https://github.com/segmentio/analytics-kotlin/blob/main/core/src/test/kotlin/com/segment/analytics/kotlin/core/platform/plugins/DestinationMetadataPluginTests.kt#L70, if you wanted to expand this